_topics/en/replace-by-fee.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Replace-by-Fee
+
+aliases:
+  - BIP125
+  - Opt-in Replace-by-Fee
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Fee Management
+  - Transaction Relay Policy
+  - Mining
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Replace by Fee (RBF)** is the ability to replace an unconfirmed
```

Comment:
'ability' doesn't seem quite right here. It makes me think that this sentence is written from the perspective of a transaction signer who is _able_ to replace their transaction. I think instead, we want this to be from the perspective of a node operator (as the second paragraph clearly is). Perhaps something like:

"**Replace by Fee (RBF)** is a node policy allowing an unconfirmed transaction in a mempool to be replaced with a different transaction that spends at least one of the same inputs and which pays a higher transaction fee."